src/app/routes/search-result/search-result.component.ts

```diff
@@ -79,8 +79,11 @@ export class SearchResultComponent implements OnInit {
         this.filteredOptions.next(options);
       });
     this.activeRoute.queryParams.subscribe(async (queryParams: any) => {
-      // Get requested claims
       try {
+        this.requestedClaims =
```

Comment:
1. You can reuse data from store, no need to make another call. It's the same list as My enrolments.
2. Calling this method every time when query params changes isn't a good idea, because list of claims won't change.
My proposition is:
1. Inject store into constructor.
2. rename `requestedClaims` into `requestedClaims$`.
3. assign `requestedClaims$ = this.store.select(OwnedEnrolmentsSelectors.getEnrolments`
4. in html change `requestedClaims` into `requestedClaims$ | async`.
$ at the end of variable name informs that it is a stream.

Reply:
thanks @dawidgil !